variables.tf

```diff
@@ -1,11 +1,154 @@
-##############################################################################
-# Input Variables
-##############################################################################
+variable "resource_group_id" {
+  description = "ID of resource group to use when creating the event stream instance"
+  type        = string
+}
 
-#variable "myvariable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = ""
-#}
+variable "es_name" {
+  type        = string
+  description = "Name of the resource instance"
+}
 
-##############################################################################
+variable "plan" {
+  type        = string
+  description = "Plan for the event streams instance : lite, standard or enterprise-3nodes-2tb"
+  default     = null
+  validation {
+    condition     = contains(["lite", "standard", "enterprise-3nodes-2tb"], var.plan)
+    error_message = "The specified plan is not a valid selection! Supported plans are: lite, standard or enterprise-3nodes-2tb."
+  }
+}
+
+variable "tags" {
+  type        = list(string)
+  description = "List of tags associated with the Event Steams instance"
+  default     = []
+}
+
+variable "region" {
+  type        = string
+  description = "IBM Cloud region where event streams will be created"
+  default     = "us-south"
+}
+
+variable "throughput" {
+  type        = number
+  description = "Throughput capacity in MB per second."
+  default     = "150" # For lite and standard plan, the allowed value of throughput is 150 MB per second. For enterprise plan it can take any value.
```

Comment:
I think the commented code here should actually be included inside the description. Otherwise consumers may not see it, as it wont be exposed in the auto generate docs.